frontend/modelarchive/src/test/java/org/pytorch/serve/archive/ModelArchiveTest.java

```diff
@@ -35,14 +39,18 @@ public void test() throws ModelException, IOException {
         // TODO Change this once we have example models on s3
         archive =
                 ModelArchive.downloadModel(
+                        URL_PATTERN_LIST,
                         modelStore,
                         "https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model");
         Assert.assertEquals(archive.getModelName(), null);
+        ModelArchive.downloadModel(
+                URL_PATTERN_LIST,
+                modelStore,
+                "https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model");
```

Comment:
This is pending since long, let's remove the use of squeezenet real model from UT and find some alternate solution for this. Maybe make it part of sanity suite/regression if required.